src/engraving/dom/measurenumber.cpp

```diff
@@ -69,7 +77,7 @@ engraving::PropertyValue MeasureNumber::propertyDefault(Pid id) const
 {
     switch (id) {
     case Pid::TEXT_STYLE:
-        return TextStyleType::MEASURE_NUMBER;
+        return style().styleV(Sid::measureNumberTextStyle);
```

Comment:
Same question here as https://github.com/musescore/MuseScore/pull/28662/files#r2190659589; isn't this override useless because `EngravingObject::propertyDefault` already does this?